pkg/terraform/gather/kubevirt/ip.go

```diff
@@ -0,0 +1,18 @@
+// Package kubevirt supply utilities to extract information from terraform state
+package kubevirt
+
+import (
+	"github.com/openshift/installer/pkg/terraform"
+)
+
+// BootstrapIP returns the ip address for bootstrap host.
+// still unsupported, because qemu-ga is not available - see https://bugzilla.redhat.com/show_bug.cgi?id=1764804
+func BootstrapIP(tfs *terraform.State) (string, error) {
```

Comment:
we can gather the IP addresses from the multus interfaces. Its true that it may change but at least we know it will be available till qemu-ga is available, if and when we will use it in the end.

Reply:
in fact, using the multus addresses is more deterministic than getting a list of addresses from the qemu-ga